main.cpp

```diff
@@ -180,10 +172,7 @@ bool Car::stop(int stopDistance)
         std::cout << "Crash!\n.";
         return false;
     }
-    else 
-    { 
-        return false;
-    }
+        return true;
```

Comment:
Just because you delete the `else { }` doesn't mean you can ignore the formatting and not fix the indentation.
- [ ] revise